cfgov/v1/wagtail_hooks.py

```diff
@@ -6,7 +6,7 @@
 from django.conf import settings
 from django.http import Http404
 from django.contrib.auth.models import Permission
-from django.utils.html import escape
+from django.utils.html import escape, format_html, format_html_join
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2344/commits/362e1236b35477c0095fd8ff5955e2e504c4c342